test/installPSRP.sh

```diff
@@ -17,10 +17,10 @@ chmod +x ./installpowershell.sh
 
 redmondpassword=$1
 isMacOS=false
-omiversion="1.1.0-28"
+omiversion="1.1.0-52"
 psrpversion="3"
-powershellDir="/opt/microsoft/powershell/6.0.0-alpha.13"
-powershellDirForMac="/usr/local/microsoft/powershell/6.0.0-alpha.13"
+powershellDir="/opt/microsoft/powershell/6.0.0-alpha.14"
+powershellDirForMac="/usr/local/microsoft/powershell/6.0.0-alpha.14"
 realdataDir="//wsscnafiler43/ostcdata$"
```

Comment:
Sorry, you are correct, you cannot link to the DFS folder with UNIX/Linux. 

Use //osfiler/ostcdata$ which is a DNS alias to //wsscnafiler43/ostcdata$ and once the link changes, I will update the alias and it should not affect your code.